.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
```suggestion
```

Also, you should only add this in your global Git config, instead of pushing a useless gitignore to every repository you touch: https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/